lib/import_govdelivery_csv.rb

```diff
@@ -26,6 +27,11 @@ def call
 
   DEFAULT_DIGEST_FREQUENCY = Frequency::IMMEDIATELY
 
+  def sanity_check_csv
+    headers = CSV.open(subscriptions_csv_path, 'r') {|c| c.first}
+    raise RuntimeError.new("Your csv files do not seem to be correct.") if headers.map(&:downcase).include?("subscriber_type")
```

Comment:
You don't need the `RuntimeError.new` here, if you `raise` a string you'll get a `RuntimeError`.

Reply:
I think it might be worth doing a sanity check on both the files. I would check that the subscriptions file has a `TOPIC_CODE` and I would check that the digests file has a `DIGEST_FOR`.